core/src/main/scala/sbtorgpolicies/libraries.scala

```diff
@@ -363,7 +372,7 @@ object libraries {
     "sbt-pgp"          -> (("com.jsuereth", "sbt-pgp", v("sbt-pgp"))),
     "sbt-release"      -> (("com.github.gseitz", "sbt-release", v("sbt-release"))),
     "sbt-site"         -> (("com.typesafe.sbt", "sbt-site", v("sbt-site"))),
-    "sbt-scalafmt"     -> (("com.geirsson", "sbt-scalafmt", v("sbt-scalafmt"))),
+    "sbt-scalafmt"     -> (("org.scalameta", "sbt-scalafmt", v("sbt-scalafmt"))),
```

Comment:
Here's the change to switch to sbt-scalafmt 2.x in projects that use sbt-org-policies.